ai_flow/scheduler_service/service/service.py

```diff
@@ -50,8 +50,10 @@ class SchedulerService(SchedulingServiceServicer):
     def __init__(self,
                  scheduler_service_config: SchedulerServiceConfig,
                  db_uri,
-                 notification_server_uri):
+                 notification_server_uri,
+                 base_log_folder):
         self._scheduler_service_config = scheduler_service_config
+        scheduler_service_config.scheduler().scheduler_config().update({'base_log_folder': base_log_folder})
```

Comment:
The scheduler_service_config contains base_log_folder config, so you can remove base_log_folder parameter.

Reply:
IMO, the `scheduler_service_config` doesn't contain the `base_log_folder` config.